Projects/DbCLI/CreateDb.py

```diff
@@ -321,144 +325,144 @@ def CreateDb(options, dataFilename='', supplier=None):
     if not mol:
       continue
 
-    if options.doPairs:
+    if args.doPairs:
       pairs = FingerprintUtils.BuildAtomPairFP(mol)
       torsions = FingerprintUtils.BuildTorsionsFP(mol)
       pkl1 = DbModule.binaryHolder(pairs.ToBinary())
       pkl2 = DbModule.binaryHolder(torsions.ToBinary())
       row = (molGuid, molId, pkl1, pkl2)
       pairRows.append(row)
-    if options.doFingerprints:
+    if args.doFingerprints:
       fp2 = FingerprintUtils.BuildRDKitFP(mol)
       pkl = DbModule.binaryHolder(fp2.ToBinary())
       row = (molGuid, molId, pkl)
       fpRows.append(row)
-    if options.doLayered:
+    if args.doLayered:
       words = LayeredOptions.GetWords(mol)
       row = [molGuid, molId] + words
       layeredRows.append(row)
-    if options.doDescriptors:
+    if args.doDescriptors:
       descrs = calc.CalcDescriptors(mol)
       row = [molGuid, molId]
       row.extend(descrs)
       descrRows.append(row)
-    if options.doPharm2D:
+    if args.doPharm2D:
       FingerprintUtils.sigFactory = sigFactory
       fp = FingerprintUtils.BuildPharm2DFP(mol)
       pkl = DbModule.binaryHolder(fp.ToBinary())
       row = (molGuid, molId, pkl)
       pharm2DRows.append(row)
-    if options.doGobbi2D:
+    if args.doGobbi2D:
       FingerprintUtils.sigFactory = Gobbi_Pharm2D.factory
       fp = FingerprintUtils.BuildPharm2DFP(mol)
       pkl = DbModule.binaryHolder(fp.ToBinary())
       row = (molGuid, molId, pkl)
       gobbi2DRows.append(row)
-    if options.doMorganFps:
+    if args.doMorganFps:
       morgan = FingerprintUtils.BuildMorganFP(mol)
       pkl = DbModule.binaryHolder(morgan.ToBinary())
       row = (molGuid, molId, pkl)
       morganRows.append(row)
 
     if not i % 500:
       if len(pairRows):
-        pairCurs.executemany('insert into %s values (?,?,?,?)' % options.pairTableName, pairRows)
+        pairCurs.executemany('insert into %(s)s values (?,?,?,?)' % args.pairTableName, pairRows)
```

Comment:
This doesn't look right at all and causes errors.
```
>>> '%(s)s' % 'test'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: format requires a mapping
```
There are many places with the same error.